### PR TITLE
azure-cli, musl-gcc, nc: add French translation; nc: update page

### PR DESCRIPTION
**Pull request by noraj**
see #13570

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/netcat.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # netcat
 
 > This command is an alias of `nc`.
+> More information: <https://manned.org/man/nc.1>.
```
Comment (spageektti) [inline pages/common/netcat.md:4]:
```suggestion
> More information: <https://manned.org/nc>.
```

File: pages.fr/common/netcat.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# netcat
+
+> Cette commande est un alias de `nc`.
+> Plus d'informations : <https://manned.org/man/nc.1>.
```
Comment (spageektti) [inline pages.fr/common/netcat.md:4]:
```suggestion
> Plus d'informations : <https://manned.org/nc>.
```

**Review by spageektti — APPROVED** [review]
Thanks!

File: pages/common/netcat.md (hunk 1 above)
Comment (spageektti) [reply pages/common/netcat.md:4]: This is the English one

Comment (noraj) [reply pages/common/netcat.md:4]: The suggested links redirects to the original one.
https://manned.org/nc ➡️ https://manned.org/man/nc.1
Also, that would mean changing the link on all other languages.

Comment (spageektti) [reply pages/common/netcat.md:4]: Some time ago we changed most of the links from manned.org/man/* to manned.org/* so we should update this too

Comment (noraj) [reply pages/common/netcat.md:4]: Ok, I'll update the link for this PR then do another PR to change all links.

**Review by spageektti — APPROVED** [review]

**Review by nicokosi — APPROVED** [review]
👍

**Pull request merged**